src/components/input-group/__styles__/index.ts

```diff
@@ -2,7 +2,7 @@ import { styled } from '@linaria/react'
 import { ElIcon } from '../../icon/__styles__'
 import { ElLabel } from '../../label/__styles__'
 import { ElInput, elHasInputError } from '../../input/__styles__'
```

Comment:
**note:** Fixing up to import from the root `textarea` folder instead of relying on internal file structures.